core/middleware/region.py

```diff
@@ -19,7 +19,7 @@ def middleware(request):
             admin_request
             or request.path.startswith(reverse("webhook"))
             or request.path.startswith("/_ah/warmup")
-            or request.path == "favicon.ico"
+            or request.path in ("/favicon.ico", "/robots.txt")
```

Comment:
Could you also add the robots.txt file (it should be accessible from /, /admin/, /admin/de/, /admin/en/ etc) and block the entire API (API + admin panel)? It should not happen but... we don't want to be visible in search engines 🤔

Reply:
Done.